mappings/net/minecraft/client/util/InputUtil.mapping

```diff
@@ -2,38 +2,57 @@ CLASS net/minecraft/class_3675 net/minecraft/client/util/InputUtil
 	FIELD field_16237 UNKNOWN_KEYCODE Lnet/minecraft/class_3675$class_306;
 	FIELD field_20333 GLFW_RAW_MOUSE_MOTION_SUPPORTED_HANDLE Ljava/lang/invoke/MethodHandle;
 	FIELD field_20334 GLFW_RAW_MOUSE_MOTION I
-	METHOD method_15981 fromName (Ljava/lang/String;)Lnet/minecraft/class_3675$class_306;
-		ARG 0 s
+	METHOD method_15981 fromTranslationKey (Ljava/lang/String;)Lnet/minecraft/class_3675$class_306;
+		ARG 0 translationKey
 	METHOD method_15983 setMouseCallbacks (JLorg/lwjgl/glfw/GLFWCursorPosCallbackI;Lorg/lwjgl/glfw/GLFWMouseButtonCallbackI;Lorg/lwjgl/glfw/GLFWScrollCallbackI;)V
 		ARG 0 handle
+		ARG 2 cursorPosCallback
+		ARG 3 mouseButtonCallback
+		ARG 4 scrollCallback
 	METHOD method_15984 setCursorParameters (JIDD)V
-	METHOD method_15985 getKeyCode (II)Lnet/minecraft/class_3675$class_306;
+		ARG 0 handler
+	METHOD method_15985 fromKeyCode (II)Lnet/minecraft/class_3675$class_306;
+		ARG 0 keyCode
+		ARG 1 scanCode
 	METHOD method_15986 setKeyboardCallbacks (JLorg/lwjgl/glfw/GLFWKeyCallbackI;Lorg/lwjgl/glfw/GLFWCharModsCallbackI;)V
 		ARG 0 handle
+		ARG 2 keyCallback
+		ARG 3 charModsCallback
 	METHOD method_15987 isKeyPressed (JI)Z
 		ARG 0 handle
+		ARG 2 code
 	METHOD method_21735 isRawMouseMotionSupported ()Z
 	METHOD method_21736 setRawMouseMotionMode (JZ)V
 		ARG 0 window
 		ARG 2 value
 	CLASS class_306 KeyCode
```

Comment:
maybe just `Key` for this class so `fromKeyCode` returning a `Key` makes sense (than returning a `KeyCode`)